src/styles/blocks/creators.scss

```diff
@@ -0,0 +1,71 @@
+.creators {
+  &__main {
+    &--title {
```

Comment:
Use a more descriptive name for the title element like '__title' instead of '__main--title'.